main/ertp/ERTP-Guide/Overview.md

```diff
@@ -0,0 +1,260 @@
+# ERTP Guide
+
+[ERTP](https://agoric.com/documentation/glossary/#ertp) (*Electronic
+Rights Transfer Protocol*)
+is Agoric's token standard for transferring tokens and other digital assets in
+JavaScript. Using the
+[ERTP API](https://agoric.com/documentation/ertp/api/),
+you can easily create and use digital assets, all of which are
+transferred exactly the same way and with exactly the same security properties. 
+
+ERTP uses
+*[object capabilities](https://agoric.com/documentation/glossary/#object-capabilities)*
+to enforce access control. If your program has a reference to an
+object, it can call methods on that object. If it doesn't have a
+reference, it can't. For more on object capabilities, see
+[Chip Morningstar's post](http://habitatchronicles.com/2017/05/what-are-capabilities/).
+
+## Objects Overview
+
+![ERTP object relationships](./assets/relationships.svg)  
+
+There are eight fundamental ERTP objects, two of which are parts of
+another, and one which is used by that same object. For each entry,
+the name is linked to its primary page in this Guide. 
+
+- **[Amount](./Amounts.md)** **[(Glossary)](https://agoric.com/documentation/glossary/#amount)**:
```

Comment:
Let's not start with Amounts, let's start with Digital Assets. Then explain that Mints are the only thing that mints them. And that they are stored in Purses and Payments. 

To explain Issuers, I think we should start from the scenario of receiving a payment from someone you don't know. An Issuer is the source of truth for whether it is a valid payment.